Source/Microsoft.Teams.Apps.DLLookup/Helpers/GraphUtilityHelper.cs

```diff
@@ -69,6 +70,7 @@ public async Task<IEnumerable<DistributionList>> GetDistributionListsAsync(strin
         {
             try
             {
+                query = Uri.EscapeDataString(query);
```

Comment:
Please inline this into the interpolated string on line 77. Do not modify the local variable `query`.

Reply:
Done, used interpolated string on line 77